priv/perf/apps/load_test/lib/test_runner.ex

```diff
@@ -0,0 +1,49 @@
+defmodule LoadTest.TestRunner do
+  alias ExPlasma.Encoding
+
+  @tests %{
+    "deposits" => {LoadTest.Runner.Deposits, LoadTest.Scenario.Deposits}
+  }
+
+  @configs %{
+    "deposits" => %{
+      token: Encoding.to_binary("0x0000000000000000000000000000000000000000"),
+      initial_amount: 500_000_000_000_000_000,
+      deposited_amount: 200_000_000_000_000_000,
+      transferred_amount: 100_000_000_000_000_000
+    }
+  }
+
+  def run() do
+    [test, rate, period] = System.argv()
```

Comment:
@boolafish Can you point out what params should have to be configured aside from rate and period? (watcher_url, ethereumex_ url etc)

Reply:
ok. it's not an  issue  since it supports the same env vars that old perf tests support